clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -277,6 +284,9 @@ static void checkTableNameIdentifer(NameIdentifier ident) {
     checkTableNamespace(ident.namespace());
   }
 
+  Namespace getTableFullNamespace(Namespace tableNamespace) {
```

Comment:
You'd better make this private if it is only used here.

Reply:
ok

Reply:
Done